styles.css

```diff
@@ -43,27 +36,6 @@ div {
   border-radius: 9px;
 }
 
```

Comment:
This comment is regarding the `h1` and `h3` global styling (and a disclaimer that this is super nitpicky)!

Because `<h*>` elements already have built in sizing, I'd consider removing both of these global styling properties! If you need the `h3` to look larger relatively to the `h1`, you could consider making it an `h2` rather than altering the global styling of `h3` to be `1.25rem`.

I've included two screenshots below to help illustrate my reasoning here- In both, I added in an `h2` with the `Which type of message` text between the `h1` and `h3`. When the `h1` and `h3` stylings are changed, there's very little size difference between your `h1` and `h2` elements! By using the built in styling, you get a nice size difference between all three elements.
![Screen Shot 2021-02-21 at 11 11 01 AM](https://user-images.githubusercontent.com/62263439/108634179-9cf01300-7435-11eb-9a74-fadf23cec05e.png)
![Screen Shot 2021-02-21 at 11 11 01 AM](https://user-images.githubusercontent.com/62263439/108634155-8053db00-7435-11eb-9fb6-fd257483184d.png)